pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -1,34 +1,96 @@
 #![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Not this PR, but we should make sure the pallet has docs and lints that follows the pattern of the other ones.